internal/handlers/handler_extended_configuration.go

```diff
@@ -11,12 +11,20 @@ type ExtendedConfigurationBody struct {
 
 	// SecondFactorEnabled whether second factor is enabled
 	SecondFactorEnabled bool `json:"second_factor_enabled"`
+
+	// TOTP Period
+	TOTPPeriod int `json:"totp_period"`
 }
 
 // ExtendedConfigurationGet get the extended configuration accessible to authenticated users.
 func ExtendedConfigurationGet(ctx *middlewares.AutheliaCtx) {
 	body := ExtendedConfigurationBody{}
 	body.AvailableMethods = MethodList{authentication.TOTP, authentication.U2F}
+	if ctx.Configuration.TOTP != nil {
+		body.TOTPPeriod = ctx.Configuration.TOTP.Period
+	} else {
+		body.TOTPPeriod = 30
```

Comment:
Use the constant defined in the validator here

Reply:
I have a feeling I can just use the value from ctx, pretty sure my if else is redundant and I didn't actually solve the problem I was intending to solve when I did it - a test was failing with vague information. I am going to check.